modules/oadp-review-backup-restore.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-review-backup-restore_{context}"]
+= Previewing resources before running backup and restore
+
+{oadp-short} backs up application resources based on the type, namespace, or label. This means that you can view the resources after the backup is complete. Similarly, you can view the restored objects based on the namespace, persistent volume (PV), or label after a restore operation is complete. To preview the resources in advance, you can do a dry run of the backup and restore operations.
+
+.Prerequisites
+
+* You have installed the OADP Operator.
+
+.Procedure
+
+. To preview the resources included in the backup before running the actual backup, run the following command:
++
+[source,terminal]
+----
+$ velero backup create <backup-name> --snapshot-volumes false # <1>
+----
+<1> Specify the value of `--snapshot-volumes` parameter as `false`.
++
+. To know more details about the backup resources, run the following command:
++
+[source,terminal]
+----
+$ velero describe backup <backup_name> --details # <1>
+----
+<1> Specify the name of the backup.
++
+. To preview the resources included in the restore before running the actual restore, run the following command:
++
+[source,terminal]
+----
+$ velero restore create --from-backup <backup-name> # <1>
```

Comment:
I will not hold up the merge over this single issue, but please consider fixing this in a later PR:
```suggestion
$ velero restore create --from-backup <backup_name> # <1>
```